hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1398,6 +1399,46 @@ private void testParquetDFSSource(boolean useSchemaProvider, List<String> transf
     testNum++;
   }
 
+  private void prepareORCDFSSource(boolean useSchemaProvider, boolean hasTransformer) throws IOException {
+    prepareORCDFSSource(useSchemaProvider, hasTransformer, "source.avsc", "target.avsc",
+            PROPS_FILENAME_TEST_ORC, ORC_SOURCE_ROOT, false);
+  }
+
+  private void prepareORCDFSSource(boolean useSchemaProvider, boolean hasTransformer, String sourceSchemaFile, String targetSchemaFile,
```

Comment:
wonder if we can simplify this test?

Reply:
Srue thing. done. thanks a lot for your review.